examples/getting-started/blocks/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This `blocks/` directory is a `workspace`, with its own `package.json`, so that the [connector plugin](https://github.com/kienstra/headless-blocks/blob/33c40fe4a22f59fe28983087c5e450e759a69acf/js/src/helpers/addPreview.ts#L9) can `import` these blocks into the block editor.